app/src/main/AndroidManifest.xml

```diff
@@ -118,6 +118,13 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/file_provider_paths"/>
         </provider>
+        <service android:name=".CardsOnPowerScreenService" android:label="@string/cards_on_power_screen"
```

Comment:
On my device this "Loyalty cards by Catima" gets cut off. Maybe just use the app name instead?

Reply:
Done

Reply:
@string/app_name please :)

Reply:
Here you go, Maintainer 🚀